src/main/features/claim/views/check-and-send.njk

```diff
@@ -23,7 +23,7 @@
           }
         )
     }}
-    {{ row('Phone:', draftClaim.claimant.mobilePhone.number, paths.claimantMobilePage.uri) }}
+    {{ row('Phone:', draftClaim.claimant.mobilePhone.number, paths.claimantMobilePage.uri, bold = true) }}
```

Comment:
Please separate these changes out into a different PR, there's over 2000 changes in this PR, any unrelated changes should be done separately